udata/tests/apiv2/test_datasets.py

```diff
@@ -81,7 +81,7 @@ def test_get_missing_param(self):
 
     def test_get_next_page(self):
         '''Should fetch 2 pages of resources from the API'''
-        resources = [ResourceFactory() for _ in range(80)]
+        resources = [ResourceFactory() for _ in range(40)]
```

Comment:
👍 probably because of https://github.com/opendatateam/udata/pull/2755/files#diff-a62c9bd3310470c9b7bfe01b30f76621a938f5d5be834d2b479dafd146b3d154R16 ?